### ISSUE TITLE
v7.2 x64 bit installer tries to install in $PROGRAMFILES

### ISSUE THREAD

**Issue by SinghRajenM** [notepad-plus-plus/notepad-plus-plus#2526]
### Description of the Issue

v7.2 x64 bit installer tries to install in $PROGRAMFILES (e.g. C:\Program Files (x86)\Notepad++\).

### Steps to Reproduce the Issue

1. Install v7.2 32 bit version
2. Now try to install v7.2 x64 bit version
3. Observe the path on install location page

### Expected Behavior

N++ should be installed in $PROGRAMFILES64 (e.g. C:\Program Files\Notepad++\).

### Actual Behavior

N++ is installed in $PROGRAMFILES (e.g. C:\Program Files (x86)\Notepad++\).

### Debug Information

**Notepad++ v7.2   (64-bit)**
Build time : Nov  2 2016 - 11:27:56
**Path : C:\Program Files (x86)\Notepad++\notepad++.exe**
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10
Plugins : mimeTools.dll NppConverter.dll 

### Additional Info

I feel moving InstallDir code (InstallDir "$PROGRAMFILES64\${APPNAME}") from nppSetup.nis to globalDef.nsh caused this isssue.

```niss
!ifdef ARCH64
InstallDir "$PROGRAMFILES64\${APPNAME}"
!else
InstallDir "$PROGRAMFILES\${APPNAME}"
!endif
```

Please refer few links address the same problem.
- http://nsis.sourceforge.net/Reference/InstallDirRegKey
- http://forums.winamp.com/showthread.php?t=284184

As per first link
> InstallDirRegKey
This attribute tells the installer to check a string in the registry, and use it for the install dir if that string is valid. **_If this attribute is present, it will override the InstallDir attribute_** if the registry key is valid, otherwise it will fall back to the InstallDir default.

So statement 
``` nsis
InstallDir "$PROGRAMFILES64\${APPNAME}"
```
in globalDef.nsh has no effect if N++ 32 bit version is already installed.

**Comment by SinghRajenM** [comment]
**Additional info:** 
This happen when 32bit is already installed and trying to install 64 bit.

**Comment by SinghRajenM** [comment]
Issue is fixed in e6fad9bd4f4031fdb7e05505d7c52ab39589853d
